CONTRIBUTING.md

```diff
@@ -158,9 +153,9 @@ to every git commit message:
 Signed-off-by: Joe Smith <joe.smith@email.com>
 ```
 
-Please use your real name, we really dislike pseudonyms or anonymous
+Please use your real name; we really dislike pseudonyms or anonymous
 contributions. We are in the open-source world without secrets. If you
-set your `user.name` and `user.email` git configs, you can sign your
+set your `user.name` and `user.email` git configs, you can sign-off your
```

Comment:
Signed commits and Sign-offs are different things. https://help.github.com/articles/signing-commits-using-gpg/

Reply:
I am aware; that's why I modified that section. It seems to me that it is referring to sign-offs and not actual PGP signings. Here is the full section for [context](https://github.com/54/gitea/blob/f298d7da7d4d808a60ee84013c5b8d7ee5745e7c/CONTRIBUTING.md#sign-your-work). It never mentions PGP keys. But, it does give an example of using the standard git sign-off.

Maybe I missed something!

Reply:
Ooh nvm, I though about `git commit -S` 😅

Reply:
@bkcsoft Fixed.